README.md

```diff
@@ -1,90 +1,98 @@
+The vast majority of the Filecoin Project conversations, including implementation, community support, ecosystem new and etc, are held on:
+
+- [Forums](#forums) (mainly on Github Discussions)
+- GitHub Issues (see [Implementation related repo](#impelmentation-related-repo) section below)
+- Chat (mainly on Slack) (see [Chat](#chat) section below)
+
+The Filecoin Project is an open sourced project with a very supportive and welcoming community, where many community members are responsive in various of forums and slack channels. We also have [Filecoin community ambassadors](https://github.com/filecoin-project/community/discussions/116#discussioncomment-617066) here to support you, tag their GitHub handlers in GitHub discussions or @fil-community-ambassadors in all slack channels if you have any questions!
+
+Implementation dev teams tend to check their corresponding GitHub repo issues and discussions regularly and try to respond within 5 business days.
 
 We also push updates to the community via:
 
+- Forum - [`Network and implementations updates`](https://github.com/filecoin-project/community/discussions/categories/network-and-implementations-updates) updates and [`news and events`](https://github.com/filecoin-project/community/discussions/categories/news-and-events)
 - Filecoin blog: [https://blog.filecoin.io](https://blog.filecoin.io)
+- CryptoComputeLab blog(proof updates): [https://research.protocol.ai/groups/cryptocomputelab/](https://research.protocol.ai/groups/cryptocomputelab/)
+- Youtube: [Filecoin](https://www.youtube.com/channel/UCPyYmtJYQwxM-EUyRUTp5DA)
 - Twitter: [@Filecoin](https://twitter.com/Filecoin)
 - WeChat ID: Filecoin-Official
 
 ![filecoin qr code](/images/qrcode_for_gh_da36751a6108_1280.jpg)
 
-Before posting to different communications channels, make sure to read the [Code of Conduct](https://github.com/filecoin-project/community/blob/master/CODE_OF_CONDUCT.md).
 
-### Community forum
+### Forums
 
-[https://github.com/filecoin-project/community/discussions](https://github.com/filecoin-project/community/discussions)
+#### Filecoin Community Forum
 
-When in doubt, please post here!
+When in doubt or curiosity, please post in Filecoin community forum](https://github.com/filecoin-project/community/discussions) !
 
 We love to hear what the community has to say, whether it is to:
 
 - Ask a question
-- Propose a new feature
+- Gathering community feedbacks on a new feature proposal before open an FIP 
 - Share a new project you're working on
 - Find collaborators for your own community project
 - And whatever else! Honestly!
 
 The discussion forum uses the same Code of Conduct as our other community channels. Please make sure to read this before posting.
 
+> Note: Chinese community forum is available [here](https://github.com/filecoin-project/community-china/discussions), which is managed by [CoinSummer](https://github.com/CoinSummer).
```

Comment:
```suggestion
> Note: A Chinese-language community forum is also available at https://github.com/filecoin-project/community-china/discussions. It is managed by [CoinSummer](https://github.com/CoinSummer).
```

Use more descriptive link text instead of "here" whenever possible, to make it easier to scan.
(Explanation: https://uxmovement.com/content/why-your-links-should-never-say-click-here/)